include/wx/font.h

```diff
@@ -216,16 +252,51 @@ class wxFontInfo
 
     // Default copy ctor, assignment operator and dtor are OK.
 
+
+    // Helper functions for converting between integer and fractional sizes.
+    static int ToIntPointSize(float pointSize) { return wxRound(pointSize); }
+    static float ToFloatPointSize(int pointSize)
+    {
+        wxCHECK_MSG( pointSize == -1.0f ||
+                     (pointSize >= 0 && static_cast<float>(pointSize) < FLT_MAX),
+                     -1, "Invalid font point size" );
```

Comment:
Why is this check written like this? `pointSize` is `int` but is compared to `-1.0f`. `static_cast` of `pointSize` is not needed. In case the check failed, simply `-1` would be returned, but for success there is another cast to `float`...

Reply:
There is definitely some confusion here but, worse, the check for `FLT_MAX` doesn't actually work neither, I don't know what was I thinking. I'll fix this, thanks.